src/PrestaShopBundle/Resources/config/routing/admin/sell/orders/carts.yml

```diff
@@ -1,3 +1,51 @@
+admin_carts_index:
+  path: /
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Order\CartController::indexAction
+    _legacy_controller: AdminCarts
+    _legacy_link: AdminCarts
+    _legacy_feature_flag: carts_index
+
+admin_carts_search:
+  path: /
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\CommonController::searchGridAction
+    gridDefinitionFactoryServiceId: prestashop.core.grid.definition.factory.cart
+    redirectRoute: admin_carts_index
+    _legacy_controller: AdminCarts
+    _legacy_link: AdminCarts
+    _legacy_feature_flag: carts_index
+
+admin_carts_delete:
+  path: /{cartId}
+  methods: [ DELETE ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Order\CartController::deleteCartAction
+    _legacy_controller: AdminCarts
+    _legacy_feature_flag: carts_index
+  requirements:
+    cartId: \d+
+
+admin_carts_bulk_delete:
+  path: /bulk-delete
+  methods: [ POST ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Order\CartController::bulkDeleteCartAction
+    _legacy_controller: AdminCarts
+    _legacy_link: AdminCarts
+    _legacy_feature_flag: carts_index
+
+admin_carts_export:
+  path: /export
+  methods: [ GET ]
+  defaults:
+    _controller: PrestaShopBundle\Controller\Admin\Sell\Order\CartController::exportCartAction
+    _legacy_controller: AdminCarts
+    _legacy_link: AdminCarts:exportcart
+    _legacy_feature_flag: carts_index
```

Comment:
as you can see you put carts_index on every action (no matter if its index, export or delete..), because once we enable index page, all the actions are enabled (and its ok). So I would say the "_index" is redundant, it should simply be enough having "carts". And we will always only have one flag per page to keep it simple :man_shrugging: